### ISSUE TITLE
Timing of tests incorrect when sequential

### ISSUE THREAD

**Issue by scottcarey** [etorreborre/specs2#507]
````scala
import org.specs2.mutable.Specification

object SpecsSpecs extends Specification {

  sequential

  "timing tests" should {
    "add up, this one takes 10 seconds" in {
      println(s"start1 ${new java.util.Date}")
      Thread.sleep(10 * 1000)
      println(s"end1 ${new java.util.Date}")
      ok
    }
    "and this one takes 9 seconds" in {
      println(s"start2 ${new java.util.Date}")
      Thread.sleep(9 * 1000)
      println(s"end2 ${new java.util.Date}")
      ok
    }
  }
}
````

Running this results in:

````
start1 Tue Sep 13 14:47:26 PDT 2016
end1 Tue Sep 13 14:47:36 PDT 2016
start2 Tue Sep 13 14:47:36 PDT 2016
end2 Tue Sep 13 14:47:45 PDT 2016
[info] SpecsSpecs
[info] 
[info] timing tests should
[info]   + add up, this one takes 10 seconds (10 seconds, 20 ms)
[info]   + and this one takes 9 seconds (9 seconds, 3 ms)
[info] 
[info] 
[info] Total for specification SpecsSpecs
[info] Finished in 10 seconds, 20 ms
[info] 2 examples, 0 failure, 0 error
[info] 
[info] Passed: Total 2, Failed 0, Errors 0, Passed 2
````

It appears the algorithm for printing out hos much time has passed is just taking the max time of each test. 

Ideally it would use the actual wall clock time before and after.

I have learned to not trust the timings reported by specs2 over time, but today noticed that the time was suspiciously equal to the longest running test in a sequential run.

**Comment by scottcarey** [comment]
This was on specs 3.8.4

**Comment by scottcarey** [comment]
Same result with 3.8.5

**Comment by etorreborre** [comment]
The fix is in `3.8.5-20160914090233-50e99f0`.

**Comment by scottcarey** [comment]
I validated the fix for the case above.

I also validated it for non-sequential cases that had the same issue -- when all examples can not execute concurrently due to the number of threads available being less than the number of examples to execute.  20 examples run in parallel, that each sleep for 5 seconds on a system with 8 available threads measured 15 seconds as expected.  Before this change it reports 5 seconds.
